### PR TITLE
Packages satysfi-class-cv.0.1.0 and satysfi-class-cv-doc.0.1.0

### PR DESCRIPTION
**Pull request by MasWag**
This pull-request concerns:
-`satysfi-class-cv.0.1.0`: SATySFi class file for Curriculum Vitae
-`satysfi-class-cv-doc.0.1.0`: Documentation of a class for CV

---
* Homepage: https://github.com/MasWag/cv-satysfi
* Source repo: git+https://github.com/MasWag/cv-satysfi.git
* Bug tracker: https://github.com/MasWag/cv-satysfi/issues

---
:camel: Pull-request generated by opam-publish v2.0.3
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-class-cv-doc.0.1.0 satysfi-class-cv.0.1.0
- [x] Add to snapshot `snapshot-develop--1` :: satysfi-class-cv-doc.0.1.0 satysfi-class-cv.0.1.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-class-cv-doc.0.1.0 satysfi-class-cv.0.1.0
- [x] Add to snapshot `snapshot-stable-0-0-6--1` :: satysfi-class-cv-doc.0.1.0 satysfi-class-cv.0.1.0

### CODE REVIEW

File: packages/satysfi-class-cv/satysfi-class-cv.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+license: "MIT"
+homepage: "https://github.com/MasWag/cv-satysfi"
+bug-reports: "https://github.com/MasWag/cv-satysfi/issues"
+dev-repo: "git+https://github.com/MasWag/cv-satysfi.git"
+depends: [
+  "satysfi" {>= "0.0.6" & < "0.0.7"}
+  "satyrographos" {>= "0.0.2.6" & < "0.0.3"}
+  "satysfi-base" {>= "1.2.1" & < "2.0.0"}
+  "satysfi-fonts-asana-math" {>= "000.958+1+satysfi0.0.4"}
+  "satysfi-parallel" {>= "0.0.1"}
```
Comment (na4zagin3) [inline packages/satysfi-class-cv/satysfi-class-cv.0.1.0/opam:19]: Can you specify the upper bound?  How about this?

```suggestion
  "satysfi-parallel" {>= "0.0.1" & < "0.1" }
```

**Comment by na4zagin3** [comment]
Sorry for my late reply.  I just added some comments.  Can you please address them?

**Comment by MasWag** [comment]
Thanks a lot. I applied your suggestions!

**Comment by na4zagin3** [comment]
Thank you!  Merging.

**Pull request merged**